pyopensprinkler/__init__.py

```diff
@@ -27,6 +27,7 @@ def __init__(self, url, password, opts=None):
         self._opts = opts
         self._programs = {}
         self._stations = {}
+        self._flow_rate = 0
```

Comment:
For consistency I think this should not be separate from `self._state`

Reply:
Can you expand on this. Not sure I understand how flow_rate would be combined with state. Do you mean as an attribute of state ?

Reply:
This is no longer needed if the code is moved into properties

Reply:
Done